ot/da.py

```diff
@@ -843,10 +927,896 @@ def predict(self, x):
 
         if self.computed:
             K = kernel(
-                x, self.xs, method=self.kernel, sigma=self.sigma, **self.kwargs)
+                x, self.xs, method=self.kernel, sigma=self.sigma,
+                **self.kwargs)
             if self.bias:
                 K = np.hstack((K, np.ones((x.shape[0], 1))))
             return K.dot(self.L)
         else:
             print("Warning, model not fitted yet, returning None")
             return None
+
+##############################################################################
+# proposal
+##############################################################################
+
+
+# adapted from sklearn
+
+class BaseEstimator(object):
+    """Base class for all estimators in scikit-learn
```

Comment:
add POt reference and a link to the original source in sk-learn

Reply:
oh yes, i remove all the scikit refs like this one